SU-HSDO/suhumsci
# SHS-8: Codeception testing private page

# READY FOR REVIEW

## Summary
Splits the private page test out to two different tests - permissions and content. The permissions test will test all roles viewing a private page. The content test will test to see if certain elements are available on the node edit form.

## Need Review By (Date)
09/07

## Urgency
low

## Steps to Test
1. run Codeception tests for permissions: ```blt codeception --group=permissions``` and verify that all tests pass.
2. run Codeception tests for content: ```blt codeception --group=content``` and verify that all tests pass.

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)
